dlgr/griduniverse/experiment.py

```diff
@@ -1024,6 +1024,15 @@ def create_network(self):
         )
         return class_(max_size=self.num_participants + 1)
 
+    def create_node(self, *args, **kwargs):
+        try:
+            return super(Griduniverse, self).create_node(*args, **kwargs)
+        finally:
+            if not self.networks(full=False):
+                # If there are no spaces left in our networks we can close
+                # recruitment, to alleviate problems of over-recruitment
+                self.recruiter.close_recruitment()
```

Comment:
I think this needs to be `self.recruiter()` in all cases to work with `dallinger/master`.

Reply:
aha. Thanks!